src/add-triage-labels.ts

```diff
@@ -0,0 +1,44 @@
+import { Probot } from 'probot';
+import { DOCUMENTATION_LABEL, SEMANTIC_PREFIXES, SEMVER_LABELS, SEMVER_NONE_LABEL } from './constants';
+import { addLabels } from './utils/label-utils';
+
+export function addBasicPRLabels(probot: Probot) {
+  probot.on(
+    [
+      'pull_request.opened',
+      'pull_request.unlabeled',
+      'pull_request.labeled',
+      'pull_request.synchronize',
+    ],
+    async context => {
+      const { pull_request: pr } = context.payload;
+
+      const getSemanticPrefix = (title: string) => title.split(':')[0];
+      
+      // Label Docs PRs as Semver-Patch PRs.
+      const isDocsPR = getSemanticPrefix(pr.title) === SEMANTIC_PREFIXES.DOCS;
+      if (isDocsPR) {
+        await addLabels(context.octokit, {
+          ...context.repo({}),
+          prNumber: pr.number,
+          labels: [
+            SEMVER_LABELS.PATCH,
```

Comment:
does this force `semver/patch` for all `docs:` PRs?

If that's the goal, this should probably remove all other `semver/*` labels too.

FWIW I don't agree that docs changes are always `semver/patch`, as many of them do not change the shipped binary.

Reply:
changed it to respect existing semver labels - it now only adds a label if there's none already present.